CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Changelog
+* Fixed a bug in the **format** command where indicatorField field.
```

Comment:
What was the bug we fixed? Was it any fix or we just added tests to validate indicator fields?